core/src/test/java/org/infinispan/stats/SingleStatsTest.java

```diff
@@ -106,7 +106,11 @@ public void testStats() {
       refreshStats();
       assertEquals(TOTAL_ENTRIES, stats.getCurrentNumberOfEntries());
       assertEquals(EVICTION_MAX_ENTRIES, stats.getCurrentNumberOfEntriesInMemory());
-      assertEquals(TOTAL_ENTRIES - EVICTION_MAX_ENTRIES, stats.getEvictions());
+      // Eviction stats with passivation can be delayed
+      eventuallyEquals((long) TOTAL_ENTRIES - EVICTION_MAX_ENTRIES, () -> {
```

Comment:
I know this is a bit hard to get correctly, but what happens if, after this call asserts true, the number of evictions changes. We should tie this to some kind of quiet state  where we can guarantee there will be no more evictions.